--- a/static/src/js/views/pivot_view.js	2018-08-17 17:08:33.394491869 -0500
+++ b/static/src/js/views/pivot_view.js	2018-08-17 17:28:04.099689510 -0500
@@ -63,6 +63,7 @@
         this.sorted_column = {};
 
         this.numbering = {};
+        this.widget = [];
     },
     willStart: function () {
         var self = this;
@@ -140,6 +141,17 @@
             //noinspection FallThroughInSwitchStatementJS
             switch (field.attrs.type) {
             case 'measure':
+                if (
+                        field.attrs.widget != 'many2one' &&
+                        field.attrs.widget != 'one2many' &&
+                        field.attrs.widget != 'many2many' &&
+                        field.attrs.widget != 'selection'
+                   ) {
+                    self.widget.push(field.attrs.widget);
+                }
+                else {
+                    self.widget.push("");
+                }
                 self.active_measures.push(name);
                 break;
             case 'col':
@@ -633,6 +645,7 @@
         var measure_types = this.active_measures.map(function (name) {
             return self.measures[name].type;
         });
+        var widgets = this.widget;
         for (i = 0; i < rows.length; i++) {
             $row = $('<tr>');
             $header = $('<td>')
@@ -643,7 +656,7 @@
             if (rows[i].indent > 0) $header.attr('title', groupby_labels[rows[i].indent - 1]);
             $header.appendTo($row);
             for (j = 0; j < length; j++) {
-                value = formats.format_value(rows[i].values[j], {type: measure_types[j % nbr_measures]});
+                value = formats.format_value(rows[i].values[j], {type: measure_types[j % nbr_measures], widget: widgets[j % nbr_measures]});
                 $cell = $('<td>')
                             .data('id', rows[i].id)
                             .data('col_id', rows[i].col_ids[Math.floor(j / nbr_measures)])
